Add basic help page

calebwoods opened the pull request:
Basic help page layout

![help_page](https://cloud.githubusercontent.com/assets/640501/13547805/1a697b10-e2ae-11e5-9960-59efb4db63dd.gif)

defnotryan commented on js/components/pages/PassagePage.react.js:
```diff
@@ -118,7 +118,7 @@ export class PassagePage extends Component {
               selectedIndex={activeIndex}
             ></PassageSelect>
-            <a href="#">Help</a>
+            <Link className="right" to="/help">Help</Link>
```
I guess we don't need this className anymore since it's now inline-flex.

defnotryan commented:
Just an extraneous className, other than that looks good to merge. FYI for anyone else looking at this PR- the actual help page looks better than in the screenshot above. The margins are a better and the headers are styled as well as the "Get Started" link at the bottom.

calebwoods commented:
Updated the styles to better match @ajcarmen's static html file.  Also added a back button so you don't have to scroll.

![updated_help](https://cloud.githubusercontent.com/assets/640501/13549621/acb4ca14-e2d7-11e5-918b-982857d0f3c0.gif)
